### ISSUE TITLE
[Bug]: Tab content turns completely black and freezes

### ISSUE THREAD

**Issue by WebworkrNet** [mozilla-mobile/fenix#20748]
### Steps to reproduce

Variant 1:
1 tab is open.

Variant 2:
Multiple tabs are open.
A tab switch follows.

### Expected behaviour

Correct display of the called web page, no freezing of the tab.

### Actual behaviour

Variant 1
While visiting the website, the tab unexpectedly turns completely black and freezes.

Variant 2:
When switching the tab, the preview image is initially displayed properly. However, once the switch is complete, the target tab is completely black and freezes .

### Device name

Samsung Galaxy XCover 4s (SM-G398FN)

### Android version

Android 10

### Firefox release type

Firefox

### Firefox version

91.1.0 (Build #2015826187) AC: 91.0.12, 1d401758fb GV: 91.0-20210804193234 AS: 79.0.0

**Comment by WebworkrNet** [comment]
Similarities to #20744

**Comment by WebworkrNet** [comment]
Possible similarities to #20733

**Comment by WebworkrNet** [comment]
![20210808_172206](https://user-images.githubusercontent.com/43509007/128637137-1ec408ff-e743-4f09-adc1-611d10ffe341.jpg)
![20210808_172254](https://user-images.githubusercontent.com/43509007/128637138-1e9bde3c-bc29-4bc2-9f6a-ad19573a5fe2.jpg)

**Comment by AlanSanchezP** [comment]
This happens to me whenever I want to open a new tab, no matter if its through a link, multi tab or direct search/paste in the URL bar.
It just sometimes works and sometimes (most of the time) gives me a blank screen that I can't refresh or do anything but close and try again

**Comment by WebworkrNet** [comment]
@AlanSanchezP The URL is displayed in the address bar so with the two screenshots?

**Comment by gaul** [comment]
I experience the same symptoms on a Lenovo Duet Chromebook:

92.0a1 (Build #2015826891)
AC: 92.0.20210808143127, 41548c2f59
GV: 92.0a1-20210808090543
AS: 81.0.1

**Comment by kbrosnan** [comment]
This will be tracked in bz-1724728

**Comment by gaul** [comment]
Note that this is not a screen flickering issue -- I cannot load or interact with any page elements except for pull-to-refresh which never completes.

**Comment by WebworkrNet** [comment]
@kbrosnan, I have to agree with @gaul there, this ticket is not about flickering. I only gave the references to show similarities. Similar does not mean identical, so I was a bit puzzled that this issue was grouped under another one. Watched this on bugzilla first though.

If I saw it correctly, the Bugzilla ticket is considered resolved and the fixes have already been incorporated into the current versions of Firefox and Firefox Nightly.

However, for this Github ticket, I cannot confirm that the problem has been fixed. The phenomenon I described continues to occur.

**Comment by WebworkrNet** [comment]
The problem also occurs under Android 11. 
In the meantime, I have upgraded my device.

**Comment by WebworkrNet** [comment]
It has been repeatedly pointed out that this is not "screen flickering".
Contrary to the status under Bugzilla this bug has not been fixed yet.
I therefore ask to reopen this issue. Thanks

**Comment by WebworkrNet** [comment]
@SoftVision-LorandJanos
Unfortunately, no one responds to my last comments. Should I open a new ticket?
